src/secondClass.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Vendor\Hw3;
+
+class secondClass extends abstractClass
+{
+    public $value = 'test poly';
```

Comment:
why this property is public? Definitely it should be private or protected and will have a setter